pkg/trustedresources/verifier/verifier.go

```diff
@@ -72,7 +73,7 @@ func FromPolicy(ctx context.Context, k8s kubernetes.Interface, policy *v1alpha1.
 		if err != nil {
 			return nil, fmt.Errorf("authority %q contains an invalid hash algorithm: %w", a.Name, err)
 		}
-		if a.Key.Data == "" && a.Key.SecretRef == nil {
+		if a.Key.Data == "" && a.Key.SecretRef == nil && a.Key.KMS == "" {
 			return nil, ErrorEmptyKey
 		}
 		if a.Key.Data != "" {
```

Comment:
clang is erroring out with a message to convert the if elses to a switch.
Please fix.

Reply:
sure, I should have fixed it

Reply:
Instead of this, the fix could be 
switch {
  case a.Key.Data != "":
         ....
  case a.Key.SecretRef != nil:
       ...
}
As suggested by clang.

Reply:
oh thanks!!